File: quasar/Quasar.DataTransfer/Enderecos/Responses/EnderecoResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Quasar.DataTransfer.Cidades.Responses;

namespace Quasar.DataTransfer.Enderecos.Responses
{
    public class EnderecoResponse
    {
        public int Codigo {get; set;}
        public int Numero {get; set;}
        public string Bairro {get; set;}
        public string Rua {get; set;}
        public string PontoReferencia {get; set;}
        public string Complemento {get; set;}
        public string Cep {get; set;}
        public CidadeResponse Cidade {get; set;}     
    }
}

File: quasar-front/src/app/shared/models/responses/endereco.response.ts
import { CidadesResponse } from 'src/app/shared/models/responses/cidades.response';


export class EnderecoResponse {
  public codigo: number;
  public numero: number;
  public bairro: string;
  public rua: string;
  public pontoReferencia: string;
  public complemento: string;
  public cep: string;
  public cidade: CidadesResponse;

  constructor(params: Partial<EnderecoResponse>) {
    this.codigo = params.codigo || 0;
    this.numero = params.numero || 0;
    this.bairro = params.bairro || "";
    this.rua = params.rua || "";
    this.pontoReferencia = params.pontoReferencia || "";
    this.complemento = params.complemento || "";
    this.cep = params.cep || "";
    this.cidade = params.cidade || new CidadesResponse({});
  }

}
